Requirements.txt

```diff
@@ -0,0 +1,32 @@
+Functionele eisen:
+
+    De applicatie moet verschillende gebruikersrollen ondersteunen, waaronder beheerders (Hella, Paulus, Robin) en personeelsleden, met elk hun specifieke rechten en toegangsniveaus.
+    Beheerders hebben toegang tot alle functies die de applicatie biedt.	
```

Comment:
Wees voorzichtig met hele algemene termen als Alle. Wat is alle? mag hij ook iets namens een personeelslid doen? en beschrijf ook waarom je een bepaalde functionaliteit nodig is. Soms is dat lastig om te doen, maar dan is het juist goed om daar over na te denken of het eigenlijk wel nodig is.